src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -918,8 +922,8 @@ internal class RecommendationService(
 
   private fun isStatusOpen(it: RecommendationEntity) =
     recommendationStatusRepository.findByRecommendationId(it.id)
-      .any { it.active && (it.name == "BOOK_TO_PPUD" || it.name == "DNTR_DOWNLOADED") }
-      .not()
+      .filter { it.active }
+      .none { (it.name == "BOOK_TO_PPUD" || it.name == "DNTR_DOWNLOADED") }
```

Comment:
Similar here I would guess too.
Is it some kind of refactor suggestion?
I think it would be a good idea to make sure that this has some really strong tests around it.

Reply:
There already is a decent - [paramaterized test](https://github.com/ministryofjustice/make-recall-decision-api/blob/defea20b5524ad07aa1714ad44cae995307d0642/src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/controller/CaseOverviewControllerTest.kt#L23) for this. It passed with both the original logic and the refactored logic here 👍